src/extension.ts

```
@@ -209,21 +238,32 @@ function toggleBazelProjectSyncStatus(doc: TextDocument) {
 					workspace
 						.getConfiguration('bazel.projectview')
 						.update('notification', false);
+				} else {
+					LOGGER.error(
+						`toggleBazelProjectSyncStatus: Unexpected value returned from showWarningMessage ${val}`
+					);
```

that should raise an error, a log message is useless and requires extra support work to investigat/handle/diagnose